Gemfile

```diff
@@ -5,5 +5,6 @@ source 'https://rubygems.org'
 # Specify your gem's dependencies in rubocop-sorbet.gemspec
 gemspec
 
+gem 'racc', require: false
```

Comment:
why this is needed here?

Reply:
```sh
bundle exec rspec

An error occurred while loading spec_helper. - Did you mean?
                    rspec ./spec/spec_helper.rb

Failure/Error: require 'rubocop'

LoadError:
  cannot load such file -- racc/parser
# ./vendor/bundle/ruby/3.3.0/gems/parser-3.1.3.0/lib/parser.rb:12:in `<top (required)>'
# ./vendor/bundle/ruby/3.3.0/gems/rubocop-ast-1.24.0/lib/rubocop/ast.rb:3:in `<top (required)>'
# ./vendor/bundle/ruby/3.3.0/gems/rubocop-ast-1.24.0/lib/rubocop-ast.rb:3:in `require_relative'
# ./vendor/bundle/ruby/3.3.0/gems/rubocop-ast-1.24.0/lib/rubocop-ast.rb:3:in `<top (required)>'
# ./vendor/bundle/ruby/3.3.0/gems/rubocop-1.40.0/lib/rubocop.rb:14:in `<top (required)>'
# ./lib/rubocop-changed.rb:3:in `<top (required)>'
# ./spec/spec_helper.rb:3:in `<top (required)>'
No examples found.
No examples found.


Finished in 0.00007 seconds (files took 0.20414 seconds to load)
0 examples, 0 failures, 1 error occurred outside of examples

Finished in 0.00007 seconds (files took 0.20414 seconds to load)
0 examples, 0 failures, 1 error occurred outside of examples
```

`racc` was removed from the Standard Library gems in Ruby 3.3 (https://stdgems.org/racc/).
Adding it does not seem to break tests on Ruby 3.2 for example, so I think it's safe.

`racc` is a transitive dependency of rubocop which is used in this gem directly in the specs, so it's a test dependency.

I realise that rubocop target version is set to 2.7, so maybe it could be a part of or automatically resolved a bigger upgrade (rubocop), but it's outside of the scope of this PR.
```
Ruby 2.7
status: eol
release date: 2019-12-25
normal maintenance until: 2022-04-01
EOL: 2023-03-31
```

Would you like me to revert the addition of `racc` to the Gemfile?